Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BUILDER_IMAGE=erlang:23.3.4.6-alpine
-ARG RUNNER_IMAGE=erlang:23.3.4.6-alpine
+ARG RUNNER_IMAGE=alpine
```

Comment:
This is just the default; you’ll need to make sure to update the same environment variable arg in the `.buildkite/scripts/make_image.sh`

Reply:
maybe you can do that?

Reply:
I did it.